juniper_codegen/src/lib.rs

```diff
@@ -549,6 +550,114 @@ pub fn graphql_scalar(attr: TokenStream, body: TokenStream) -> TokenStream {
         .into()
 }
 
+/// `#[derive(GraphQLScalarValue)]` macro for deriving a [`ScalarValue`]
```

Comment:
Then let's name it `#[derive(ScalarValue)]` directly. There is no sense stopping us from that.